plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -110,9 +110,9 @@ class Bazel(
         super.run(
             args = args,
             workingDir = workingDir,
-            // Disable the optional wrapper script under `tools/bazel`, to ensure the --version option works.
+            // Disable the optional wrapper script under `tools/bazel` only for the "--version" call.
```

Comment:
> can be relevant by defining

I don't understand this. Do you mean "can be relevant e.g. in case a specific '.bazelrc'
depending on the value of an environment variable is defined"?

Reply:
I tried to explain the setup of our customer project and the phrasing is quite difficult...

They have in their project a wrapper script in `tools/bazel`. This script reads an environment variable and, depending on its value, change the .bazelrc of the project.
Therefore when you call `bazel mod graph`, this script "setups" your build environment depending on this environment variable.
However, bazel uses this script for all commands even if you call `bazel version`. In that case, the script fails because it does not support the `--version` parameter. While the correct fix would be to change the wrapper script to support this parameter, this is not possible to force all the projects to do so.
And, in all cases, ORT must make sure that the wrapper script is enabled when `mod graph` is called, as it may contain some build changing logic as described above.